src/main/java/org/broadinstitute/consent/http/db/mapper/UsersAndCardsReducer.java

```diff
@@ -0,0 +1,82 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.ArrayList;
+import org.broadinstitute.consent.http.enumeration.RoleStatus;
+import org.broadinstitute.consent.http.models.LibraryCard;
+import org.broadinstitute.consent.http.models.Institution;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.UserRole;
+import org.jdbi.v3.core.mapper.MappingException;
+import org.jdbi.v3.core.result.LinkedHashMapRowReducer;
+import org.jdbi.v3.core.result.RowView;
+
+public class UsersAndCardsReducer implements LinkedHashMapRowReducer<String, User> {
```

Comment:
maybe this is better defined as `CardsWithUserReducer`? The primary point being to collect library cards and users, if they exist.